docs/reference/objects/product/variant/index.md

```diff
@@ -19,6 +19,13 @@ object
 
 #### Attributes
 
+## `variant.booking_fee_rate`
+{: .d-inline-block }
+number
+{: .label .fs-1 }
+
```

Comment:
Should we mention that this fallback to the company booking fee rate if not present?

Reply:
I don't think that needs to be mentioned here to be honest.

Reply:
Maybe you right. I was thinking that if no fee is defined on the variant, one might think that this return nil or something, but from a theme developer that might be irrelevant